src/Microsoft.AspNetCore.Razor.Sdk/SDK/Sdk.targets

```diff
@@ -0,0 +1,20 @@
+<!--
+***********************************************************************************************
+Sdk.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="$(_MicrosoftAspNetCoreRazorSdkCurrentVersionPath)" 
+    Condition="'$(_MicrosoftAspNetCoreRazorSdkCurrentVersionPath)' != '' AND Exists('$(_MicrosoftAspNetCoreRazorSdkCurrentVersionPath)')" />
+
+  <Import Project="$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.CurrentVersion.targets"
+    Condition="'$(MicrosoftAspNetCoreRazorSdkTargetsImported)' != 'true' AND Exists('$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.CurrentVersion.props')" />
```

Comment:
Shouldn't `MicrosoftAspNetCoreRazorSdkTargetsImported` be a private property? Seems like an implementation detail of ours. Also kind of odd that we then expect the currentversion.targets to set this to `true`.

Reply:
There's some precedence to using a non-private property - https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.Common.props#L137.

Reply:
I think it's fine to make it public. If it's a contract between our package and our SDK, I think it should be public. We need to have this discipline to get these things right (most of the time).